src/app/containers/ArticleTimestamp/timeFormatTests/generateTimeFormats.js

```diff
@@ -1,7 +1,7 @@
 import fs from 'fs';
 import path from 'path';
 import { timestampsFixtures, format } from './testUtils';
-import services from '#server/utilities/serviceConfigs';
+import services from '../../../../server/utilities/serviceConfigs';
```

Comment:
```suggestion
import services from '#server/utilities/serviceConfigs';
```

Reply:
the babel-node transpilation fails because of the dir alias 😞

Reply:
Oh right, I see.